Samples/Samples/View/CalendarEventPage.xaml

```diff
@@ -9,58 +9,63 @@
              x:DataType="essentials:Event"
              Title="Event"
              Padding="20,20,20,20">
+    <views:BasePage.Resources>
+        <ResourceDictionary>
+            <x:String x:Key="DateDisplayFormatter">{0:dd/MM/yy hh:mm tt}</x:String>
+        </ResourceDictionary>
+    </views:BasePage.Resources>
     <views:BasePage.Content>
-      <StackLayout>
-          <Label Text="{Binding Title}"
+        <StackLayout>
+            <Label Text="{Binding Title}"
                  FontSize="34"
                  TextColor="Black"
                  HorizontalTextAlignment="Center"/>
-          <Grid>
-              <Grid.ColumnDefinitions>
-                  <ColumnDefinition/>
-                  <ColumnDefinition/>
-                  <ColumnDefinition Width="Auto"/>
-                  <ColumnDefinition/>
-              </Grid.ColumnDefinitions>
-              <Grid.RowDefinitions>
-                  <RowDefinition Height="Auto"/>
-                  <RowDefinition Height="Auto"/>
-                  <RowDefinition Height="Auto"/>
-                  <RowDefinition/>
-              </Grid.RowDefinitions>
-              <Label Grid.Row="1"
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition/>
+                    <ColumnDefinition/>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition/>
+                </Grid.RowDefinitions>
+                <Label Grid.Row="1"
```

Comment:
Formatting: Alignment of the attributes seems to be off - it might be github way of presenting it on the screen. If so, please ignore this comment. Otherwise we should make sure that the formatting is correct.

Reply:
Have used code maid to clean up the page, but perhaps at a later date code maid settings will have to be altered as they seem to like to keep their xaml parameters on one line.